site/content/en/docs/Guides/Client SDKs/local.md

```diff
@@ -158,6 +158,34 @@ chmod o+r gameserver.yaml
 docker run --network=host --rm -v $(pwd)/gameserver.yaml:/tmp/gameserver.yaml us-docker.pkg.dev/agones-images/release/agones-sdk:{{<release-version>}} --local -f /tmp/gameserver.yaml
 ```
 
+If you would like to run the ClientSDK and your game server container together with Docker Compose create a `docker-compose.yaml` file.
```

Comment:
Do we want to add an extra note here that's something like...

```suggestion
if you run Docker on a OS that doesn't run Docker natively or in a VM, such as on Windows or macOS., you may want to to run the ClientSDK and your game server container together with Docker Compose . To do so, create a `docker-compose.yaml` file setup with a network overlay shared between them:
```

Just so people have a _why_ on they might want to do this.

WDYT?

Reply:
I like, makes sense. Updated in latest commit.